controllers/controllers.js

```diff
@@ -32,3 +33,11 @@ exports.getUsers = (req, res, next) => {
     res.status(200).send({ users: response });
   });
 };
+
+exports.getArticles = (req, res, next) => {
+  fetchArticles().then((response) => {
+    response.sort((a, b) => (a.created_at > b.created_at ? 1 : -1));
```

Comment:
Please handle all sorting behaviours using SQL: this will make it much easier to add in additional sorting behaviours on later tickets, and it separates the concerns nicely too. Our controller shouldn't be doing any data manipulation beyond actually creating the response object to send, it should be receiving the data in the correct form already from the model.